### PR TITLE
Configuration footer component commit

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

File: src/app/shared/components/qbd/configuration/configuration-step-footer/configuration-step-footer.component.html [hunk 1]
```diff
@@ -1 +1,20 @@
...
+        </button>
+    </div>
+    <div>
+        <button *ngIf="ctaText !== 'Saving'" pButton type="button" class="p-button-raised" (click)="saveChanges()" [disabled]="isButtonDisabled">
+            {{ ctaText }}
+            <i class="tw-pl-10-px tw-pt-2-px !tw-text-12-px pi pi-arrow-right"></i>
+        </button>
+        <button *ngIf="ctaText === 'Saving'" pButton type="button" class="p-button-raised" disabled>
+            Saving
+            <div style="border-top-color:transparent"
```
Comment (ashwin1111) [inline src/app/shared/components/qbd/configuration/configuration-step-footer/configuration-step-footer.component.html:16]: why style as well as class?

File: src/app/shared/components/qbd/configuration/configuration-step-footer/configuration-step-footer.component.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.backBtn {
+    @apply tw-py-12-px ;
```
Comment (ashwin1111) [inline src/app/shared/components/qbd/configuration/configuration-step-footer/configuration-step-footer.component.scss:2]: same line

File: src/app/shared/components/qbd/configuration/configuration-step-footer/configuration-step-footer.component.scss [hunk 3]
```diff
@@ -0,0 +1,12 @@
+.backBtn {
```
Comment (ashwin1111) [inline src/app/shared/components/qbd/configuration/configuration-step-footer/configuration-step-footer.component.scss:1]: please don't use camel case for css class names. it should be `.back-btn`

File: src/app/shared/components/qbd/configuration/configuration-step-footer/configuration-step-footer.component.html (hunk 1 above)
Comment (DhaaraniCIT) [reply src/app/shared/components/qbd/configuration/configuration-step-footer/configuration-step-footer.component.html:16]: this is due to the spinner, in spinner we are making the top border alone transparent, so it look like a spinner

Comment (ashwin1111) [reply src/app/shared/components/qbd/configuration/configuration-step-footer/configuration-step-footer.component.html:16]: why not add it in class?

Comment (DhaaraniCIT) [reply src/app/shared/components/qbd/configuration/configuration-step-footer/configuration-step-footer.component.html:16]: shall I put like this `tw-border-b-white tw-border-r-white tw-border-l-white tw-border-t-transparent`

Comment (ashwin1111) [reply src/app/shared/components/qbd/configuration/configuration-step-footer/configuration-step-footer.component.html:16]: sure, you can add it in class

**Pull request merged**